internal/processors/hcgp/config.go

```diff
@@ -16,10 +16,17 @@
 package hcgp
 
 type Config struct {
-	ModulePath  string                 `json:"modulePath"`
-	InitOptions map[string]interface{} `json:"initOptions,omitempty"`
+	ModulePath  string      `json:"modulePath"`
+	InitOptions InitOptions `json:"initOptions,omitempty"`
```

Comment:
What do you think about:

```suggestion
	InitOptions json.RawMessage `json:"initOptions,omitempty"`
```

In this way, we can remove the custom UnmarshalJSON